wetterdienst/cli.py

```diff
@@ -198,74 +205,71 @@ def run():
         return
 
     # Sanity checks.
-    if options.readings and options.format == "geojson":
+    if (options.readings or options.forecasts) and options.format == "geojson":
```

Comment:
That should be called `observations` instead of `readings`.

```suggestion
    if (options.observations or options.forecasts) and options.format == "geojson":
```